src/icons/BackChevron.tsx

```diff
@@ -13,11 +16,11 @@ function BackChevron({ color, height }: Props) {
       height={height}
       width={height && height / 2}
       viewBox="0 0 8 16"
-      fill="none"
       xmlns="http://www.w3.org/2000/svg"
       testID="BackChevron"
     >
-      <Path
+      <AnimatedPath
+        // @ts-ignore
```

Comment:
What's the purpose of ts-ignore here?

Reply:
Typescript yells at you if that isn't present and makes you remove all the properties. Adding that ts-ignore is based on what is already done in the 'Times' component [here](https://github.com/valora-inc/wallet/blob/main/src/icons/Times.tsx#L24)

Reply:
Do we know why TS doesn't like these params? If it's a known limitation, can we use `ts-expect-error` and also add a comment on why we expect error. I don't think we should be adding generic `ts-ignore`s, those can hide real issues in the future.

Reply:
Yeah I think it's because we were using the package `svgs` instead of `react-native-svg`. I'll update the `Times` component to use that package instead in a separate PR

Reply:
Seems like `svgs` is just a wrapper around `react-native-svg` and was surprised why this all of a sudden succeeded. Dig a little more digging on this and turns out we override ts declarations for svgs here: https://github.com/valora-inc/wallet/blob/44b94f0fb450156749872d240ce5f1e875968338/src/index.d.ts#L12
Git blame says this line is from the initial commit, probably this was added when this package didn't have type definitions. We should remove this.
Also, I wonder if we should consider switching to just `react-native-svg` throughout. Don't see any value in using `svgs` since everything in this repo is react-native.